src/main/docker/jre/Dockerfile

```diff
@@ -43,4 +44,5 @@ FROM stage1
 COPY --from=stage2 /wd /
 
 # Set JRE Home
-ENV JRE_HOME=/etc/alternatives/jre_21_openjdk
+ARG JAVA_VERSION
```

Comment:
Do you actually need this second `ARG JAVA_VERSION`?

Reply:
Yes its needed inside each stage its used in.

Reply:
Ok.  Let's move setting `JRE_HOME` up to build stage 1 then so we don't need to repeat it.

Reply:
Done